docs/src/man/faq.md

```diff
@@ -2,8 +2,13 @@
 
 ### I am having issues, where do I leave a bug report?
 
-Please leave bug reports either [at the discussion board](https://discourse.julialang.org)
-or [at the Juno.jl Github repository](https://github.com/JunoLab/Juno.jl/issues).
+Please leave bug reports either at
+[Julia's discussion forum under the `Tooling ▶ Juno` category](https://discourse.julialang.org/c/tools/juno/l/latest)
+or [the Juno.jl GitHub repository](https://github.com/JunoLab/Juno.jl/issues).
```

Comment:
Mind switching the order of those? I'd rather have bug reports on GitHub than Discourse, I think.

Reply:
Yeah I prefer GitHub for having bug reports as well. Do you want not to even mention Discourse here as well ?

Reply:
Actually I want to have issue reports only on Juno.jl repo or slack for lightweight questions. But I know Discourse is a good to place to announce and some people want to report issues on there.
Just let me know your preference, @pfitzseb :)

Reply:
Just switch them around, I think. Discourse will be used no matter what we say here ;)